README.md

```diff
@@ -1,30 +1,99 @@
-# Configuration for Halide buildbots
+# Python environment
 
-To get started:
-
-Install Python 3.6+, python3-pip, and python3-venv, then run:
+To get started setting up _either_ a worker or a master, install Python 3.6+, python3-pip, and python3-venv, then create
+a virtual environment and install our Python dependencies:
```

Comment:
Ignorant q here: why do we need a venv for this? (i.e., do we *need* one, or is it just good hygiene?)

Reply:
Good hygiene, mostly. You could pip install the requirements globally, but then it might interfere with _other_ Python things running on that computer. I absolutely always use a virtualenv because it's just more convenient.